Apps/phcofensetriage/cofensetriage.json

```diff
@@ -170,6 +170,7 @@
         {
             "action": "get threat indicators",
             "description": "Retrieve the subjects, senders, domains, URLs, or MD5 or SHA256 hashes that operators identified in Cofense Triage as threat indicators within a specified timeframe",
+            "verbose": "The cef_mapping parameter is a JSON dictionary represented as a serialized JSON string, such as the result of json.dumps(). Each key in the dictionary is a potential key name in an artifact that is to be renamed to the value. For example, if the cef_mapping is {\"website\":\"requestURL\"} your artifact will have requestURL cef fields in place of website cef fields.",
```

Comment:
was this supposed to be the verbose for the `cef_mapping` parameter rather than the action?

same for the other newly added verbose action descriptions. they sound like verbose parameter descriptions

Reply:
Hey Ron, if we add verbose for the action parameters, then we are not able to see that text anywhere in the app documentation or in the tooltip. In order to make it available in the app documentation, either we add that additional details in the parameter description or in the action verbose by mentioning the parameter name.

Reply:
Oh shoot, you're right. I don't know why I was thinking verbose in parameter descriptions was a thing. In that case it makes sense to keep the old long parameter descriptions as they were so that they show up as part of the parameter information for the appropriate parameter.

Reply:
Okay, cool. We will revert the last commit.